.ci/daily/Jenkinsfile

```diff
@@ -293,7 +222,7 @@ pipeline {
         axes {
           axis {
             name 'DB'
-            values 'postgresql_96', 'mariadb_103'
+            values 'postgresql_96', 'postgresql_94', 'postgresql_107', 'postgresql_112', 'postgresql_122', 'cockroachdb_201', 'mariadb_100', 'mariadb_102', 'mariadb_103', 'mariadb_galera', 'mysql_57', 'oracle_11', 'oracle_12', 'oracle_18', 'oracle_19', 'db2_105', 'db2_111', 'sqlserver_2017', 'sqlserver_2019'
           }
           axis {
             name 'PROFILE'
```

Comment:
Let's rename the `check-sql` profile to `sql-scripts` to be more close to the current job name. Could be in a separate ticket.

Reply:
upgrade-db -> upgrade-database ?

Reply:
I agree on both. I updated them here, and will adjust the labels in scope of CAM-12866 in the shared library.